crates/ruma-client-api/src/backup.rs

```diff
@@ -118,6 +118,10 @@ pub struct EncryptedSessionData {
 
     /// First 8 bytes of MAC key, encoded in base64.
     pub mac: Base64,
+
+    /// MAC of the key, encoded in base64
+    #[serde(rename = "org.matrix.msc4048.mac2")]
```

Comment:
This field needs to be feature flagged behind a feature called `unstable-msc4048`.